components/org.wso2.carbon.identity.remotefetch.core/src/test/java/org/wso2/carbon/identity/remotefetch/core/util/RemoteFetchConfigurationUtilsTest.java

```diff
@@ -48,18 +65,21 @@ public IObjectFactory getObjectFactory() {
     @Test
     public void testParseConfiguration() throws RemoteFetchCoreException {
 
-        mockStatic(IdentityUtil.class);
-        when(IdentityUtil.getProperty("RemoteFetch.FetchEnabled")).thenReturn("true");
-        RemoteFetchConfigurationUtils.parseConfiguration();
-        assertNotNull(RemoteFetchConfigurationUtils.parseConfiguration());
-        assertTrue(RemoteFetchConfigurationUtils.parseConfiguration().isEnableCore());
+        when(IdentityUtil.getProperty(RemoteFetchConfigurationUtils.REMOTE_FETCH_ENABLED)).thenReturn("true");
+        when(IdentityUtil.getProperty(RemoteFetchConfigurationUtils.REMOTE_FETCH_WORKING_DIRECTORY)).thenReturn("tmp");
+
```

Comment:
Please remove this new line if it's unnecessary.

Reply:
These spaces were intentionally kept for logical block separation and better readability.

Reply:
ACK